src/Book/Book.UnitTests/Application/InMemoryBookDB.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Book.Infrastructure;
+using Microsoft.EntityFrameworkCore;
+
+namespace Book.UnitTests.Application;
+
+public class InMemoryBookDB
+{
+    public readonly BookContext context;
+
+    public InMemoryBookDB()
+    {
+        DbContextOptionsBuilder<BookContext> builder = new();
+        builder.UseInMemoryDatabase(databaseName: "Book");
+
+        DbContextOptions<BookContext> dbContextOptions = builder.Options;
+        context = new BookContext(dbContextOptions);
```

Comment:
Быть может, если мы нигде более не используем `dbContextOptions`, сразу от него и избавиться.
```suggestion
        context = new BookContext(builder.Options);
```